### DISCUSSION TITLE
Releasing the discussions section (September 2023)

### DISCUSSION THREAD

**Discussion by teivah** [teivah/100-go-mistakes#35] [Announcements]
Hello all,

I'd like to give this space a try. For example, people could raise questions, discuss options, opinions, etc.

Feel free to participate. 😄

**Comment by Delta456** [comment]
I sent a typo to be fixed in your twitter DM. I hope that is fixed by now!

**Reply by teivah** [reply]
I noted it, but typos, following Manning policy, will be taken into account in the second edition of the book (if it is released one day).
